src/js/utils/mimetypes.js

```diff
@@ -17,7 +17,17 @@ export const MimetypesKind = {
   mp3: 'audio/mpeg',
   aac: 'audio/aac',
   oga: 'audio/ogg',
-  m3u8: 'application/x-mpegURL'
+  m3u8: 'application/x-mpegURL',
+  bmp: 'image/bmp',
```

Comment:
do we need the ones like bmp, icon, tif, tiff? I'd vote to keep just jpg, jpeg, png, svg, webp

Reply:
Sure, that's fine. I don't imagine people would use those, but wanted to cover common image formats.